node/node.go

```diff
@@ -115,14 +123,16 @@ func New(cfg *Config, version string) (*Node, error) {
 	if n.cfg.EthNode == "" {
 		n.log.Warnw("Ethereum node address not found; will not verify against L1")
 	} else {
-		ethNodeURL, err := url.Parse(n.cfg.EthNode)
+		var ethNodeURL *url.URL
+		ethNodeURL, err = url.Parse(n.cfg.EthNode)
```

Comment:
why the changes here? and similarly on L134 and L144

Reply:
Linter started complaining for some reason. Probably a bug in it's parsing logic.